barcodes_gs1_label/models/options_config.py

```diff
@@ -0,0 +1,65 @@
+# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
+# Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class BarcodesGS1LabelOptionsConfig(models.Model):
+    _name = "barcodes.gs1.label.options.config"
+
+    name = fields.Char(string="Label Config", required=True, translate=True)
+
+    default = fields.Boolean()
+
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        required=True,
+        default=lambda x: x.env.company,
+        readonly=True,
+    )
+
+    @property
+    def BARCODE_TYPE_SELECTION(self):
+        return [
+            ("gs1-128", "GS1-128"),
+            ("gs1-datamatrix", "GS1-Datamatrix"),
+            ("ean13-code128", "EAN13 + Code128"),
+            ("ean13", "EAN13"),
+        ]
+
+    barcode_type = fields.Selection(
+        string="Barcode type",
+        selection=lambda x: x.BARCODE_TYPE_SELECTION,
```

Comment:
```suggestion
        selection=lambda self: self.BARCODE_TYPE_SELECTION,
```
Just a best practice